docs/ops/doc/examples/mesh_viz.rst

```diff
@@ -61,13 +61,13 @@ The following script accepts the binary dataset as its sole input, and creates t
         mesh = ops.op("geom.marchingCubes").input(mask).apply()
         println("mesh = ${mesh} [${mesh.triangles().size()} triangles, ${mesh.vertices().size()} vertices]")
 
-        meshVolume = ops.op("geom.size").input(mesh).apply().getRealDouble()
```

Comment:
@elevans @ctrueden let's debate whether this causes a breakage in backwards compatibility. On one hand, this line of code will no longer work unless we write a wrapper in this repo around it (and the whole point of this PR is to get rid of those), which would lead to it being a breaking change. This is pretty convincing for me, but playing devil's advocate, the apply() will return you an `Object`, and so you are violating the return type contract i.e. on paper we didn't do anything wrong.

I still lead towards this being a breaking change that requires a major version bump, though.